src/components/Popups/__tests__/PopupDrawingToolbar-test.tsx

```diff
@@ -27,14 +29,32 @@ describe('PopupDrawingToolbar', () => {
         canComment: true,
         canRedo: false,
         canUndo: false,
+        drawnPathGroups: [],
         onDelete: jest.fn(),
         onRedo: jest.fn(),
         onReply: jest.fn(),
         onUndo: jest.fn(),
         reference: { getBoundingClientRect: () => getDOMRect() },
     });
-    const getWrapper = (props?: Partial<Props>): ShallowWrapper =>
-        shallow(<PopupDrawingToolbar {...getDefaults()} {...props} />);
+
+    const popupMock = { popper: { update: jest.fn() } };
+
+    const getWrapper = (props = {}): ReactWrapper => mount(<PopupDrawingToolbar {...getDefaults()} {...props} />);
+
+    beforeEach(() => {
+        jest.spyOn(React, 'useEffect').mockImplementation(f => f());
```

Comment:
We should either spyOn/mock the React methods and use shallow or not mock them and use mount.